ci/auto-pr/create_pull_requests

```diff
@@ -32,16 +32,29 @@ function cherry_pick_and_create_pull_request () {
   git cherry-pick --no-rerere-autoupdate -m1 $commit_sha
   git push origin $new_branch
   git status
-  gh pr create --assignee $assignee \
-    --base "$branch" \
-    --title "Backport to branch($branch) : $pull_request_title" \
-    --body "Backport of $pull_request_url"
+  if [[ -z $assignee ]]; then
+    gh pr create \
```

Comment:
I'm wondering, if no one is assigned to a PR or an issue, who will notice it? One possible idea is to assign it to the person who merged the original PR. What do you think?

Reply:
Sounds a good idea. I'll try it.

Reply:
@brfrn169 Fixed it in https://github.com/scalar-labs/scalardb/pull/898/commits/ea9b8b4d1785f8b056d1007814017e0b6825e255